config/software/ruby-windows-devkit.rb

```diff
@@ -15,36 +15,36 @@
 #
 
 name "ruby-windows-devkit"
-default_version "4.5.2-20111229-1559"
+default_version "4.7.2-20130224"
 
 dependency "ruby-windows"
 
-version "4.5.2-20111229-1559" do
-  source url: "http://cloud.github.com/downloads/oneclick/rubyinstaller/DevKit-tdm-32-#{version}-sfx.exe",
-         md5: "4bf8f2dd1d582c8733a67027583e19a6"
-end
+Omnibus.logger.add(Omnibus::Logger::LEVELS.index('DEBUG'), "#{name}") { ": windows_arch_i386 is " + windows_arch_i386?.to_s }
 
-version "4.7.2-20130224-1151" do
-  source url: "http://cdn.rubyinstaller.org/archives/devkits/DevKit-mingw64-32-#{version}-sfx.exe",
-         md5: "9383f12958aafc425923e322460a84de"
+if windows_arch_i386?
+  version "4.7.2-20130224" do
+    source url: "http://cdn.rubyinstaller.org/archives/devkits/DevKit-mingw64-32-#{version}-1151-sfx.exe",
```

Comment:
ugh, wow. I assumed the certs would be good on their end but ruby/omnibus would have issues downloading because of the aforementioned stuff. Now that I look around a bit, I see we're downloading other things via SSH, so we must actually be configuring SSL correctly for our ruby installation in the cookbooks that create build boxes.